packages/synthetix-main/test/integration/modules/CollateralModuleConfiguration.test.ts

```diff
@@ -6,7 +6,7 @@ import { ethers } from 'hardhat';
 
 import { bootstrap } from '../bootstrap';
 
-describe('CollateralManagerConfiguration (SCCP)', function () {
+describe.skip('CollateralManagerConfiguration (SCCP)', function () {
```

Comment:
Now you'll see a bunch of skips added.

I simply removed all only's from AccountModule reviewed tests, and added skip's to everything else.